src/Fractions/Fraction.cs

```diff
@@ -11,51 +12,59 @@ namespace Fractions;
 /// The data type is not capable to store NaN (not a number) or infinite.
 /// </summary>
 [TypeConverter(typeof(FractionTypeConverter))]
-[StructLayout(LayoutKind.Sequential)]
+// [StructLayout(LayoutKind.Sequential)] // TODO do we need a particular order?

```

Comment:
This is important if the data type is to be transferred via marshaling (P/Invoke). However, there is no (Windows) API that could handle this data type.